FordLabs/PeopleMover
# Fix eslint errors/warnings for unit tests

### What was done:
- [x] Fixed all eslint errors and warnings for unit test files

### How to test:
1. In the package.json file, replace the lint command script with `"lint": "eslint src/**/*.test.tsx",`
2. Run that command and ensure that there are no eslint errors or warnings for unit test files.
